section-6/exercises/denysArraysMethod.js

```diff
@@ -0,0 +1,138 @@
+// Exercise 1:
+// Use forEach() to double each number in the array and print the result.
+const nums = [2, 4, 6, 8];
+nums.forEach(num => {
+    console.log(num * 2);
+});
+// One more
+nums.forEach(doubleNumbers)
+
+function doubleNumbers(number) {
+    console.log(number * 2);
+}
+
+// Exercise 2:
+// Use forEach() to print only the even numbers from an array.
+const numbers = [3, 10, 7, 8, 15, 2];
+const evenNumbers = [];
+numbers.forEach(num => {
+    if (num % 2 === 0) {
+        evenNumbers.push(num);
+    }
+});
+console.log(evenNumbers);
+
+
+console.log("Exercise 3")
+// User filter() to create an array of names that is longer than 4 characters.
+const names = ["Anna", "Jonathan", "Lee", "Michael"];
+const filteredNames = names.filter(name => name.length > 4);
+console.log(filteredNames);
+
+//Just try to use as function
+function filterByNames(array, length) {
```

Comment:
Name of function say me that you make filter inside function by name. Better naming will be filterNamesByLength, filterUserNamesByLength. It gives more info about what you do inside function

Reply:
Function arguments should also be informative. Example: (userNames, minNameLength)